test/community/assortativity.jl

```diff
@@ -7,14 +7,14 @@ using Random, Statistics
         @test @inferred assortativity(wheel_graph(n)) ≈ -1/3
     end
     @testset "Directed ($seed)" for seed in [1, 2, 3], (n, ne) in [(14, 18), (10, 22), (7, 16)]
-        g = erdos_renyi(n, ne; is_directed=true, seed=seed)
+        g = erdos_renyi(n, ne; is_directed=true, rng=RNG, seed=seed)
```

Comment:
```suggestion
        g = erdos_renyi(n, ne; is_directed=true, rng=StableRNG(seed))
```
otherwise iterating over `seed` has no purpose.

Reply:
It does, since when the `rng` and `seed` leads to seeding the `rng` with this value.